src/main/scala/org/constellation/gossip/GossipService.scala

```diff
@@ -57,7 +58,17 @@ abstract class GossipService[F[_]: Parallel, A](
       )
       sequenceToRetry = sequence.dropWhile(node => !failureNode.contains(node))
       clients <- sequenceToRetry.toList.traverse(getClientMetadata)
-      _ <- clients.traverse(spreadFn(_, message)) // TODO: Force trigger peer healthcheck if it fails
+      _ <- clients.parTraverse { client =>
```

Comment:
Will parTraverse process every call in parallel? Shouldn't we maybe use same parallelit level like 2 or 4 to not run too many calls at the same time?

Reply:
With parTraverseN

Reply:
Yep, I think it will use one thread per task, within the current `ContextShift`'s thread pool. Is there an easy way of limiting the number of threads we want to use to 2 or 4? We could use some custom `ContextShift` but I'm not sure if that's such a good way.

Reply:
> With parTraverseN

Awesome, I will try that out 👍